base/setup/lib/muifonts.h

```diff
@@ -169,7 +168,6 @@ MUI_SUBFONT ChineseSimplifiedFonts[] =
     { L"Times New Roman TUR,162",      L"Times New Roman,162" },
     { L"Tms Rmn",                L"Times New Roman" },
     { L"Trebuchet MS",           L"Droid Sans Fallback" },
-    { L"Ubuntu",                 L"Droid Sans Fallback" },
```

Comment:
Can you double-check, that things look "ok" (as good or as bad) when you install ROS in either one of chinese, japanese or korean, with those themes enabled?

Reply:
Yes they will not be changed because
MUI_SUBFONT ChineseSimplifiedFonts[] =
...
    { L"Tahoma",                 L"Droid Sans Fallback" },

and
MUI_SUBFONT JapaneseFonts[] =
    { L"Tahoma",                 L"Droid Sans Fallback" },

and
MUI_SUBFONT KoreanFonts[] =
    { L"Tahoma",                 L"Droid Sans Fallback" },

I cannot test that, because the "Droid Sans Fallback" font gets copied into the iso only for releases, and not for master builds. But like I wrote in the description already: They won't be changed by the PR!